packages/tpetra/core/src/Tpetra_Experimental_BlockCrsMatrix_def.hpp

```diff
@@ -2950,10 +2950,10 @@ class GetLocalDiagCopy {
     auto numPacketsPerLIDHost = numPacketsPerLID.view_host(); // we will modify this
     {
       typedef Impl::BlockCrsReducer<Impl::BlockCrsRowStruct<size_t>,host_exec> reducer_type;
-      const auto policy = Kokkos::RangePolicy<host_exec>(0, numExportLIDs);
+      const auto policy = Kokkos::RangePolicy<host_exec>(size_t(0), numExportLIDs);
```

Comment:
I don't think this does what you think it does.  RangePolicy will deduce the default index type on its own, not from the arguments.  We're not doing C++17-style argument type deduction.